js/src/Ice/ConnectionI.js

```diff
@@ -1065,14 +1041,14 @@ export class ConnectionI {
     }
 
     sendHeartbeat() {
-        if (this._state == StateActive || this._state == StateHolding) {
+        if (this._state === StateActive || this._state === StateHolding || this._state === StateClosing) {
```

Comment:
The "core" of this PR: keep sending heartbeats in StateClosing, but stop when we reach StatePendingClosing (= after we've sent the CloseConnection message).